docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+services:
+  skynest-db:
+    image: mysql:latest
+    container_name: skynest-db
+    ports:
+      - $DB_PORT_HOST:$DB_PORT_CONTAINER
+    restart: unless-stopped
+    env_file:
+      - db.env
+    volumes:
+      - db:/var/lib/mysql
+
+  skynest-db-flyway:
```

Comment:
This one is not needed at all. Pls remove it and let flyway run everytime via application start